### PR TITLE
Bump version 1.26.0

### PR DESCRIPTION
**Pull request by EtienneM**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ### To be Released
 
+### 1.26.0
+
 * feat(db-commands) Add `database-enable-feature` and `database-disable-feature` for database addons [#807](https://github.com/Scalingo/cli/pull/807)
```
Comment (Soulou) [inline CHANGELOG.md:7]: Nothing about windows then?

Comment (EtienneM) [reply CHANGELOG.md:7]: As it's an internal change we didn't added an entry. Do you want me to add one?

Comment (EtienneM) [reply CHANGELOG.md:7]: Moreover it does not seem to actually fix the issue...

Comment (Soulou) [reply CHANGELOG.md:7]: It depends if it would fix any issue or not

Comment (EtienneM) [reply CHANGELOG.md:7]: That's the odd thing. On @QuentinEscudierScalingo Windows computer, it seems it fixes the issue. On a Outscale Windows server, it does not fix the issue. So.... 🤷🏾

Comment (Soulou) [reply CHANGELOG.md:7]: Arf..

**Review by Soulou — APPROVED** [review]

**Pull request merged**